modules/phase_field/include/kernels/CoupledCoefReaction.h

```diff
@@ -0,0 +1,59 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#ifndef COUPLEDCOEFREACTION_H
+#define COUPLEDCOEFREACTION_H
+
+#include "CoupledReaction.h"
+#include "JvarMapInterface.h"
+#include "DerivativeMaterialInterface.h"
+
+// Forward Declaration
+class CoupledCoefReaction;
+
+template<>
+InputParameters validParams<CoupledCoefReaction>();
+
+// This kernel adds to the residual a contribution of -L*v where L is a material
```

Comment:
You should use Doxygen style comments, otherwise this will not end up in the online code docs